pyEpiabm/pyEpiabm/place.py

```diff
@@ -0,0 +1,89 @@
+#
+# Place Class
+#
+import typing
+from .person import Person
+from .place_type import PlaceType
+
+
+class Place:
+    """Creates a place class which represents spaces such
+    as cafes, restaurants and hotels where people may come
+    into contact with others outside their household.
+    """
+    def __init__(self, loc: typing.Tuple[float, float],
+                 place_type: PlaceType, cell, microcell):
+        """Constructor method.
+
+        :param loc: (x,y) coordinates of the place
+        :type loc: tuple
```

Comment:
I think it is generally good practice to specify the type of elements in an iterable like a tuple - I assume they are floats here?